src/main/fc/fc_msp.c

```diff
@@ -2284,8 +2284,9 @@ static mspResult_e mspFcProcessInCommand(uint16_t cmdMSP, sbuf_t *src)
 
 #ifdef USE_NAV
     case MSP_SET_NAV_POSHOLD:
-        if (dataSize >= 13) {
+        if (dataSize >= 15) {
```

Comment:
@breadoven let's not make changes to the MSP protocol. Settings can be used to change that and AFAIR Advanced Tuning page was already redesigned to do so

Reply:
I assumed the dataSize was equal to the total size of the variables in bytes and since 2 bytes (16 bit) have been added it needs increasing in size. Or are you saying this is automatically handled at compile time and by Configurator ?

Reply:
OK, re. stronnag's comment, had a feeling there might be some smart way of handling this. I'll remove this change ... keeps things simple at least where MSP is concerned.